src/views/authenticated/List.tsx

```diff
@@ -16,17 +19,47 @@ export function List({ data: unfilteredListItems }: Props) {
 		);
 	}, [searchTerm, unfilteredListItems]);
 
+	const handleAddItemsClick = () => {
+		navigate("/manage-list");
+	};
+
 	return (
 		<>
 			<p>
 				Hello from the <code>/list</code> page!
 			</p>
 
+			{unfilteredListItems.length === 0 && (
```

Comment:
praise: I love the short eval instead of a bunch of if-elses. it's easiery for me to follow personally.